dotcom-rendering/src/web/components/MediaMeta.tsx

```diff
@@ -126,7 +126,7 @@ export const MediaMeta = ({
 				hasKicker={hasKicker}
 			/>
 			&nbsp;
-			{!!mediaDuration && (
+			{mediaDuration !== undefined && (
```

Comment:
What does a 0 media duration mean?

Reply:
I guess you'd expect it to _not_ be 0, but don't necessarily check for it, in the same way you wouldn't expect a timestamp to be 0. Maybe we should be more explicit and check though?

Reply:
I think it's a valid state for the duration of some audio or a video to be zero.